firebase-installations/src/androidTest/java/com/google/firebase/installations/FisAndroidTestConstants.java

```diff
@@ -36,4 +36,5 @@
 
   public static final long TEST_CREATION_TIMESTAMP_1 = 2000L;
   public static final long TEST_CREATION_TIMESTAMP_2 = 2000L;
```

Comment:
what's the point of having TIMESTAMP_1 and TIMESTAMP_2 if they're the same?

Reply:
Good catch. I missed it. Updated.